post-patchcore/message/html/action/quote.js

```diff
@@ -11,7 +11,7 @@ exports.create = (api) => {
   return nest('message.html.action', function quote (msg) {
     return h('a', {
         href: '#',
-      'ev-click': () => api.app.sync.goTo({ action: 'quote', key: msg.key, value: msg.value })
+      'ev-click': (ev) => { ev.preventDefault(); api.app.sync.goTo({ action: 'quote', key: msg.key, value: msg.value }) }
```

Comment:
I might need to check how this behaves with `app.sync.locationId` (which makes unique ids for locations)